docs(instances): Fix unreachable IPv6 after routed ip migration (int-add-instances) (#2408)
scaleway/docs-content: compute/instances/troubleshooting/fix-unreachable-ipv6-rhel-based-instance.mdx

---
meta:

>   title: Fix unreachable IPv6 RHEL based Instance after transition to routed ip

Reviewer: ```suggestion
  title: Fix unreachable IPv6 RHEL-based Instances after transitioning to routed IP
```

>   description: This page helps recover or avoid unreachable IPv6 Instance after transition to routed ip

Reviewer: ```suggestion
  description: This page helps recover or avoid unreachable IPv6 Instances after transitioning to routed IP
```

content:

>   h1: Fix unreachable IPv6 RHEL based Instance after transition to routed ip

Reviewer: ```suggestion
  h1: Fix unreachable IPv6 RHEL-based Instances after transitioning to routed IP
```

>   paragraph: This page helps recover or avoid unreachable IPv6 Instance after transition to routed ip

Reviewer: ```suggestion
  paragraph: This page helps recover or avoid unreachable IPv6 Instances after transitioning to routed IP
```

tags: centos-stream rockylinux almalinux network-manager ipv6 routed ip
dates:
  validation: 2023-11-16
  posted: 2023-11-16
categories:
  - compute
---

> RHEL based Instances created before November 15th, 2023 require a NetworkManager configuration that enable correct IPv6 configuration after a transition to routed IP is performed. If your Instance is configured only with an IPv6 address, it will become unreachable after the migration. It is important to add the NetworkManager configuration so IPv6 configuration remains correct.<br />

Reviewer: ```suggestion
RHEL-based Instances created before November 15th 2023 require a NetworkManager configuration that enables correct IPv6 configuration, after a transition to routed IP is performed. If your Instance is configured only with an IPv6 address, it will become unreachable after the migration. It is important to add the NetworkManager configuration so IPv6 configuration remains correct.<br />
```

The affected O/S distributions are:

- CentOS Stream versions 8 and 9
- RockyLinux versions 8 and 9
- AlmaLinux versions 8 and 9
- Fedora versions 37 and 38

<Message type="note">
  CentOS 7.9 is not concerned by this issue as it does not support routed IP.
</Message>

## Your Instance has not been migrated to routed IP yet

Connect to your Instance and execute the following command as `root` user to add the required NetworkManager option:
```
cat << EOF > /etc/NetworkManager/conf.d/00-scaleway.conf
[connection]
# The value 0 stands for eui64 -- see nm-settings-nmcli(5)
ipv6.addr-gen-mode=0
EOF
```

> ## Your Instance has been migrated to routed IP

Reviewer 2: ```suggestion
## Your Instance has been moved to routed IP
```

> If your Instance has been migrated to routed IP and you have IPv4 connectivity add the NetworkManager configuration as outlined in the previous section. After the next reboot you will recover IPv6 connectivity while IPv4 configuration will remain untouched.

Reviewer 2: ```suggestion
If your Instance has been moved to routed IP and you have IPv4 connectivity add the NetworkManager configuration as outlined in the previous section. After the next reboot you will recover IPv6 connectivity while IPv4 configuration will remain untouched.
```

> If your Instance has been migrated to routed IP and you have IPv4 connectivity add the NetworkManager configuration as outlined in the previous section. After the next reboot you will recover IPv6 connectivity while IPv4 configuration will remain untouched.

Reviewer: ```suggestion
If your Instance has been migrated to routed IP and you have IPv4 connectivity, add the NetworkManager configuration as outlined in the previous section. After the next reboot, you will recover IPv6 connectivity while IPv4 configuration will remain untouched.
```

> If your Instance has been moved to routed IP and you have IPv4 connectivity add the NetworkManager configuration as outlined in the previous section. After the next reboot you will recover IPv6 connectivity while IPv4 configuration will remain untouched.

Reviewer 2: ```suggestion
If your Instance has been moved to routed IP and you have IPv4 connectivity, add the NetworkManager configuration as outlined in the previous section. After the next reboot, you will recover IPv6 connectivity while IPv4 configuration will remain untouched
```

### You only have IPv6 configured so you cannot connect to the Instance anymore

[!WARNING]
Two reboots of your Instance will be required in order to recover IPv6 connectivity

> If you are in this situation, you need to temporarily add an IPv4 address to connect to your Instance in order do add the NetworkManager configuration.

Reviewer: ```suggestion
If you are in this situation, you need to temporarily add an IPv4 address to connect to your Instance in order to add the NetworkManager configuration.
```
